synfig-studio/src/gui/states/state_bone.cpp

```diff
@@ -490,6 +493,9 @@ StateBone_Context::~StateBone_Context()
 
 	get_work_area()->reset_cursor();
 
+	// Turn off Active Bone rendering
+	//get_work_area()->set_active_bone_render_toggle(false);
```

Comment:
Can be removed? ^_^

Reply:
Noo! I actually need it! I did a mistake! Wrong commit! 😅